AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -205,6 +205,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75106
+solotakus-tv.ml#%#//scriptlet("adjust-setTimeout", "", "1000", "0.05")
```

Comment:
This rule works good, but I think we can speed it up and make a bit more specific, please change to:
`solotakus-tv.ml#%#//scriptlet("adjust-setTimeout", "#counter", "", "0.02")`

Reply:
Done.